nautobot/core/tests/test_templatetags_helpers.py

```diff
@@ -322,3 +322,14 @@ def test_hyperlinked_object_target_new_tab(self):
             helpers.hyperlinked_object_target_new_tab(location, "foo"),
             f'<a href="/dcim/locations/{location.pk}/" title="An important location" target="_blank" rel="noreferrer">{location!s}</a>',
         )
+
+    def test_dbm(self):
+        self.assertEqual(
+            helpers.dbm(12),
+            "12 dBm",
+        )
+        self.assertEqual(
+            helpers.dbm(-85),
+            "-85 dBm",
+        )
+        self.assertEqual(helpers.dbm(None), '<span class="text-muted">&mdash;</span>')
```

Comment:
Nitpick

```suggestion
        self.assertEqual(helpers.dbm(None), helpers.placeholder(None))
```

Reply:
I was torn between these and saw it done both ways in the tests.